mvi/learning/algorithm.py

```diff
@@ -0,0 +1,19 @@
+from abc import ABC, abstractmethod
```

Comment:
Remove, this was a premature idea. Possibly can add back in the future once more learning algorithms come into play. Since we only have PPO so far, we don't need abstraction just yet.